src/context/FormEventActions/CrmSaveMode.ts

```diff
@@ -0,0 +1,4 @@
+export enum CrmSaveMode {

```

Comment:
I think we should use the enum provided by @types/xrm. There are more than just 2 options.